be/app/src/main/java/com/deckbop/app/controllers/HelloController.java

```diff
@@ -0,0 +1,12 @@
+package com.deckbop.app.controllers;

```

Comment:
This Controller was used in the tutorial and can be removed eventually, but leaving in for now to demo